src/gameRestart.js

```diff
@@ -0,0 +1,15 @@
+import { Console } from '@woowacourse/mission-utils';
+
+export default async function gameRestart() {
+	try {
+		const ANSWER = await Console.readLineAsync('게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.\n');
+		if (ANSWER === '1' || ANSWER === '2') {
+			return ANSWER;
+		} else {
+			throw new Error('[ERROR] 1과 2외의 다른 값을 입력했습니다.');
+		}
```

Comment:
```suggestion
		if (ANSWER === '1' || ANSWER === '2') {
			return ANSWER;
		}
		throw new Error('[ERROR] 1과 2외의 다른 값을 입력했습니다.');
```

위에서 조건문을 만나게 되면 ANSWER를 return하고 나가기 때문에 else문 없이도 만약 위 if문을 만나지 않았다면 자연스럽게 throw문을 만나게 됩니다!